.gitignore

```diff
@@ -11,3 +11,6 @@ build/
 
 # Directory created by dartdoc
 doc/api/
+
+# VS Code settings
+.vscode/
```

Comment:
Side note - editor specific ignores are better handled with a global gitignore than in every repository.

Reply:
thanks for the tip!